core/src/main/scala/org/apache/spark/storage/StorageLevel.scala

```diff
@@ -172,6 +173,7 @@ object StorageLevel {
     case "NONE" => NONE
     case "DISK_ONLY" => DISK_ONLY
     case "DISK_ONLY_2" => DISK_ONLY_2
+    case "DISK_ONLY_3" => DISK_ONLY_3
     case "MEMORY_ONLY" => MEMORY_ONLY
     case "MEMORY_ONLY_2" => MEMORY_ONLY_2
     case "MEMORY_ONLY_SER" => MEMORY_ONLY_SER
```

Comment:
Don't need MEMORY_AND_DISK_3 or MEMORY_AND_DISK_SER_3?

Reply:
Thank you for review. For the question, No, @viirya . We can consider this as a kind of replacement of HDFS. This PR is not aiming to support `MEMORY_AND_DISK_3` or `MEMORY_AND_DISK_SER_3`.

Reply:
Got it. Thanks.